### ISSUE TITLE
Not able to run the program in order to refactor Maven project

### ISSUE THREAD

**Issue by pedroagribeiro** [cal101/AutoRefactor#3]
Hey there!

First a bit of context: I have set up a docker running ubuntu in order to run this program. I have a series of Maven projects that I want to refactor using this tool. I have made an alias (`autorefactor_cli`) in order to make it easier to run the program and also to make it possible to run it from anywhere within the docker.

I'm trying to run the program using the following command:

`autorefactor_cli apply --project <path_project_dir> --refactorings [TryWithResourceFactoring]`

Just to make sure that it works. When I run like this I get a FileNotFoundException, which made me wonder: "Do I have to provide the absolute path to each of the Java files?". I tried that and it didn't work since I got an error saying that it failed to read the project description.

To sum it up: my question is basically what am I doing wrong here? Is it not possible to run the tool in Maven Projects that were not developed using Eclipse? Do I have to somehow turn the Maven project into some sort of Eclipse Project? Am I using the program wrong?

Thanks in advance,

Hope you're well and healthy

**Comment by pedroagribeiro** [comment]
I just noticed that the answer to my question is in the README. Forgive for my sins :D
